src/SWP/Bundle/CoreBundle/Command/CreateUpdateOrganizationCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher MultiTenancyBundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Command;
+
+use SWP\Bundle\CoreBundle\Model\OrganizationInterface;
+use SWP\Bundle\MultiTenancyBundle\Command\CreateOrganizationCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CreateUpdateOrganizationCommand extends CreateOrganizationCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+        $this
+            ->setDescription('Creates a new organization or updates existing one.')
```

Comment:
maybe split it into two different commands as there are two operations here, `create` and `update`.